src/mgmt/srv.c

```diff
@@ -35,6 +35,9 @@
 #include "srv_internal.h"
 #include <signal.h>
 
+/** Fully qualified path to daos_server socket */
+char *ds_mgmt_server_socket_path;
```

Comment:
Is the mgmt module going to be making dRPC client calls into the daos_server? If so, we should probably start thinking of how to centralize the client connections if we have multiple modules independently trying to talk as clients on the same socket. If we're just listening for calls from daos_server into daos_io_server, though, this isn't needed -- the IO server has a listener thread that already knows its listening socket.

Reply:
nothing springs to mind as most information will probably be requested from daos__server. maybe events in the future?

Reply:
Might be best to remove in the short-term then.